mpv/scripts/run_websocket_server.lua

```diff
@@ -3,9 +3,16 @@
 
 local utils = require("mp.utils")
 
+local platform
```

Comment:
This can be simplified

```suggestion
local platform = mp.get_property_native("platform")
```